pkg/reconciler/common/images.go

```diff
@@ -41,34 +41,80 @@ var (
 	delimiter             = "/"
 )
 
+// ImageTransformer is an interface for transforming images passed to the ResourceImageTransformer
+type ImageTransformer interface {
+	HandleImage(containerName, parent string, useDefault bool) (string, bool)
+	HandleImagePullSecrets(imagePullSecrets []corev1.LocalObjectReference, log *zap.SugaredLogger) []corev1.LocalObjectReference
```

Comment:
Handle -> Transform?

Handle makes me think it's a 'visit' function more than something returning another value.

Alternatively: a doc would work